src/profile_service/profile.js

```diff
@@ -1,141 +1,143 @@
 'use strict';
 
 const constants = require('../yoti_common/constants');
-const { Attribute } = require('../data_type/attribute');
-
-module.exports.Profile = class Profile {
-  constructor(profileData) {
-    this.profileData = Object.assign({}, profileData);
-  }
+const { BaseProfile } = require('./base.profile');
 
+/**
+ * Profile of a human user with convenience methods to access well-known attributes.
+ *
+ * @class Profile
+ */
+class Profile extends BaseProfile {
   /**
+   * The full name attribute.
+   *
    * @returns {null|Attribute}
    */
   getFullName() {
     return this.getAttribute(constants.ATTR_FULL_NAME);
   }
 
   /**
+   * Corresponds to secondary names in passport, and first/middle names in English.
+   *
    * @returns {null|Attribute}
    */
   getGivenNames() {
     return this.getAttribute(constants.ATTR_GIVEN_NAMES);
   }
 
   /**
+   * Corresponds to primary name in passport, and surname in English.
+   *
    * @returns {null|Attribute}
    */
   getFamilyName() {
     return this.getAttribute(constants.ATTR_FAMILY_NAME);
   }
 
   /**
+   * Date of birth.
+   *
    * @returns {null|Attribute}
    */
   getDateOfBirth() {
     return this.getAttribute(constants.ATTR_DATE_OF_BIRTH);
   }
 
   /**
+   * Did the user pass the age verification check?
+   *
    * @returns {null|Attribute}
    */
   getAgeVerified() {
     return this.getAttribute(constants.ATTR_AGE_VERIFIED);
   }
 
   /**
+   * Corresponds to the gender in the passport; will be one of the strings "MALE", "FEMALE"
```

Comment:
I looked this up in the wiki:
```suggestion
   * Corresponds to the gender in the passport; will be one of the strings "MALE", "FEMALE", "TRANSGENDER"
```

Looks like we need to update in the other SDKs too

Reply:
Corrected in 09a6cc7